pkg/storage/memory/memory.go

```diff
@@ -69,6 +69,7 @@ type MemoryBackend struct {
 	maxTuplesPerWrite             int
 	maxTypesPerAuthorizationModel int
 	mu                            sync.Mutex
+	currentTime                   storage.TimeProvider
```

Comment:
```suggestion
	timeProvider                   storage.TimeProvider
```

Reply:
Obsolete as we don't have the time provider anymore